src/pages/ProjectListPage/ProjectListPage.tsx

```diff
@@ -1,22 +1,104 @@
+import { ChangeEvent, useEffect, useState } from "react"
+import { useInView } from "react-intersection-observer"
+import { useLocation, useNavigate } from "react-router-dom"
+
 import { Box } from "@chakra-ui/react"
+import { Container, Stack, useMediaQuery } from "@chakra-ui/react"
+
+import { useQueryClient } from "@tanstack/react-query"
+
+import ProjectFilter from "@components/ProjectFilter/ProjectFilter"
+import ProjectList from "@components/ProjectList/ProjectList"
 
-import ProjectListSection from "@pages/HomePage/components/ProjectListSection/ProjectListSection"
+import { useAllProjectQuery } from "@pages/HomePage/hooks/queries/useAllProjectQuery"
+import { SortSelectType } from "@pages/HomePage/types/type"
 
-import ResultInfo from "./components/ResultInfo"
-import SearchSection from "./components/SearchSection"
+import { QUERYKEY } from "@constants/queryKey"
+
+import ResultInfo from "./components/ResultInfo/ResultInfo"
+import SearchBarSection from "./components/SearchBarSection/SearchBarSection"
 
 const ProjectListPage = () => {
   const params = new URLSearchParams(window.location.search)
```

Comment:
```suggestion
const ProjectListPage = () => {
  const params = useSearchParams()[0]
```
[P3] : 요기를 react router의 `useSearchParams`를 써주셔야돼요!
이렇게해야 location.search가 바뀌어도 바로 반영될 수 있습니다!